modules/readme/update_related_references.py

```diff
@@ -0,0 +1,92 @@
+import os
+import json
+import subprocess
+from github import Github
+import requests
+from iteration_utilities import unique_everseen
+from ruamel.yaml import YAML
+
+GH_TOKEN = os.environ["GH_TOKEN"]
+GH_ORG_NAME = os.getenv("GH_ORG_NAME", "cloudposse")
+GH_SEARCH_PATTERN = os.getenv("GH_SEARCH_PATTERN", "terraform-aws-eks")
```

Comment:
You'll want to compute this dynamically by looking at the first 3 fields of this module's github repo name. 

e.g. a repo like `https://github.com/cloudposse/terraform-aws-vpc-peering-multi-account`

the first 3 fields are `terraform-aws-vpc` and the pattern would be `terraform-aws-vpc-.*`